samples/wifi/thread_coex/README.rst

```diff
@@ -292,8 +265,14 @@ Testing
 	|               |             | on client** is seen on the DUT nRF7002 DK's UART console window.        |
 	+---------------+-------------+-------------------------------------------------------------------------+
 
-#. Observe that the Wi-Fi throughput result appears on the test PC terminal on which **iperf** server is run.
-   The Thread throughput result appears on the minicom terminal connected to the nRF7002 DK, on which Thread is run in the client role.
+When the sample runs Wi-Fi UDP throughput in client mode, a peer device (test PC) runs UDP throughput in server mode using the following command:
```

Comment:
Does the user need to run that command or does it just happen?
If the user needs to run it, then rather:
"... runs UDP throughput in server mode.
Use the following command:"